# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1498334543 14400
#      Sat Jun 24 16:02:23 2017 -0400
# Node ID 315a3dcc229c6b3bd869d657c080151b0037e919
# Parent  93371ce5378d3d210fe3a01a27f90f1e626e3be8
allow make dist to work again

* liboctave/external/slatec-fn/module.mk (liboctave_EXTRA_DIST):
Delete variable.

diff --git a/liboctave/external/slatec-fn/module.mk b/liboctave/external/slatec-fn/module.mk
--- a/liboctave/external/slatec-fn/module.mk
+++ b/liboctave/external/slatec-fn/module.mk
@@ -39,13 +39,9 @@ EXTERNAL_SOURCES += \
   %reldir%/r9lgic.f \
   %reldir%/xdbetai.f \
   %reldir%/xdgami.f \
   %reldir%/xdgamit.f \
   %reldir%/xgmainc.f \
   %reldir%/xsgmainc.f \
   %reldir%/xbetai.f
 
-liboctave_EXTRA_DIST += \
-  %reldir%/derfc.in.f \
-  %reldir%/erfc.in.f
-
 DIRSTAMP_FILES += %reldir%/$(octave_dirstamp)
